client/asset/eth/multirpc.go

```diff
@@ -1059,18 +1059,22 @@ func (m *multiRPCClient) withAll(
 ) error {
 
 	var atLeastOne bool
-	var errs []error
-	for _, p := range m.nonceProviderList() {
+	var errs []string
+	providers := m.nonceProviderList()
+	for i := range providers {
+		p := providers[i]
```

Comment:
`for i, p := range providers {`

Reply:
`p` is a pointer, I think there used to be an issue with using pointer range var with previous versions of go yh?

Reply:
Not, but if you maintain a reference to it outside of its iteration that's problematic.